RecoEgamma/EgammaElectronProducers/python/lowPtGsfElectronFinalizer_cfi.py

```diff
@@ -0,0 +1,45 @@
+import FWCore.ParameterSet.Config as cms
+
+from RecoEgamma.EgammaTools.regressionModifier_cfi import regressionModifier106XUL
+
+lowPtRegressionModifier = regressionModifier106XUL.clone(
+    modifierName = 'EGRegressionModifierV3',
+    rhoTag = 'fixedGridRhoFastjetAll',
+    eleRegs = dict(
+        ecalOnlyMean = dict(
+            lowEtHighEtBoundary = 20.,
+            ebLowEtForestName = ":lowPtElectron_eb_ecalOnly_05To50_mean",
+            ebHighEtForestName = ":lowPtElectron_eb_ecalOnly_05To50_mean",
+            eeLowEtForestName = ":lowPtElectron_ee_ecalOnly_05To50_mean",
+            eeHighEtForestName = ":lowPtElectron_ee_ecalOnly_05To50_mean",
+            ),
+        ecalOnlySigma = dict(
+            lowEtHighEtBoundary = 20.,
+            ebLowEtForestName = ":lowPtElectron_eb_ecalOnly_05To50_sigma",
+            ebHighEtForestName = ":lowPtElectron_eb_ecalOnly_05To50_sigma",
+            eeLowEtForestName = ":lowPtElectron_ee_ecalOnly_05To50_sigma",
+            eeHighEtForestName = ":lowPtElectron_ee_ecalOnly_05To50_sigma",
+            ),
+        epComb = dict(
+            ecalTrkRegressionConfig = dict(
+                lowEtHighEtBoundary = 20.,
+                ebLowEtForestName = ":lowPtElectron_eb_ecalTrk_05To50_mean",
+                ebHighEtForestName = ":lowPtElectron_eb_ecalTrk_05To50_mean",
+                eeLowEtForestName = ":lowPtElectron_ee_ecalTrk_05To50_mean",
+                eeHighEtForestName = ":lowPtElectron_ee_ecalTrk_05To50_mean",
+                ),
+            ecalTrkRegressionUncertConfig = dict(
+                lowEtHighEtBoundary = 20.,
+                ebLowEtForestName = ":lowPtElectron_eb_ecalTrk_05To50_sigma",
+                ebHighEtForestName = ":lowPtElectron_eb_ecalTrk_05To50_sigma",
+                eeLowEtForestName = ":lowPtElectron_ee_ecalTrk_05To50_sigma",
+                eeHighEtForestName = ":lowPtElectron_ee_ecalTrk_05To50_sigma",
+                ),
+        )
+    ),
+)
+
+lowPtGsfElectrons = cms.EDProducer("LowPtGsfElectronFinalizer",
```

Comment:
normally the _cfi file name should match the module defined in it. I suggest to git mv this to `lowPtGsfElectrons_cfi.py`

Also, once the `LowPtGsfElectronIDProducer::fillDescriptions` is updated to inject a valid default config, please clone from it here.

Reply:
> Also, once the LowPtGsfElectronIDProducer::fillDescriptions is updated to inject a valid default config, please clone from it here.

I'm confused by this request. Do you mean IDProducer or Finalizer? If the latter, please see my comments here https://github.com/cms-sw/cmssw/pull/32391#discussion_r541454001

Reply:
right, I meant the Finalizer